packages/carbon-graphs/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # ChangeLog
 
+* Changed
+  * Added a condition to check if graphContainer is present and then resize the graph.
```

Comment:
Should it move to the below Unreleased section?

Reply:
I saw Pranav's PR and added it in the same place. Please let me know if it should be moved.

Reply:
I saw one of Pranav's PR. It was added below the Unreleased section. Correct me if I'm wrong

![image](https://user-images.githubusercontent.com/62761486/99818166-7effd300-2b13-11eb-8a11-bb6f6d77e5dd.png)

https://github.com/cerner/terra-graphs/pull/18/files?short_path=7b73f4b#diff-7b73f4b73c95577a16e8a58d2e1377b2204811d315c020884e80b9f22128c42a

Reply:
Yes, I would say it needs to be in the unreleased section as well since we're no longer releasing after every PR.

Reply:
I would also move it to below `Migrated to the terra-graphs mono-rep.` so that this is in chronological order.

Reply:
I have updated it [here](https://github.com/JananiGunasekaran/terra-graphs/commit/510ef795df968f9d5b6f4ea8393ac2e2e17cfe1e).